### PR TITLE
RENO-3736: Fix Blog LinkCard padding

### PR DESCRIPTION
**Pull request by isastettler**
[Jira Ticket](https://jira.nypl.org/browse/RENO-3736)

## **This PR does the following:**

- Adds a new prop called `isBlog` to TextFormatted component
- Adds a conditional `marginBottom="l"` to TextFormatted when `isBlog` is true
- passes `isBlog` to TextFormatted in CardList

### Review Steps:

- [ ] Pull in branch `git fetch origin RENO-3736/fix-blog-bug-padding-link-card`
- [ ] Switch to relevant brach `git checkout RENO-3736/fix-blog-bug-padding-link-card`
- [ ] Start a local build `npm run build && npm start`
- [ ] Confirm unit tests are passing `npm test`
- [ ] Confirm cypress tests are passing `npm run build && npm start` and `npm run cypress` in separate terminals

### Front End Review Steps:

- [ ] View [Example Blog](https://scout-git-reno-3736-fix-blog-bug-padding-link-card-nypl.vercel.app/blog/2020/05/29/manhattanhenge-reading-list)
- [ ] Confirm that the CardList description has a marginBottom
- [ ] View[Example Section Front](https://scout-git-reno-3736-fix-blog-bug-padding-link-card-nypl.vercel.app/education)
- [ ] Confirm there is no extra `marginBottom` applied to description texts

### CODE REVIEW

File: src/components/shared/ContentComponents/CardList.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ function CardList({ id, type, title, description, items }: CardListProps) {
   return (
     <Box id={`${type}-${id}`} mb="xl">
       {title && <Heading level="two" text={title} />}
-      {description && <TextFormatted html={description} />}
+      {description && <TextFormatted html={description} isBlog />}
```
Comment (nypl-wluisi) [inline src/components/shared/ContentComponents/CardList.tsx:46]: I would remove the `isBlog` prop and just wrap `TextFormatted` like this:

```jsx
<Box mb="l">
  <TextFormatted html={description} />
</Box>
```

**Review by nypl-wluisi — CHANGES_REQUESTED** [review]

File: src/components/shared/TextFormatted/TextFormatted.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ interface TextFormattedProps {
 }
 
 function TextFormatted({ html }: TextFormattedProps) {
+  console.log("formattedText used!");
```
Comment (nypl-wluisi) [inline src/components/shared/TextFormatted/TextFormatted.tsx:9]: Remove the `console.log`

**Review by nypl-wluisi — CHANGES_REQUESTED** [review]
@isastettler Looks good, just 1 more small change.

File: src/components/shared/TextFormatted/TextFormatted.tsx (hunk 2 above)
Comment (isastettler) [reply src/components/shared/TextFormatted/TextFormatted.tsx]: Thanks for catching this.

**Pull request merged**